src/main/java/org/sonarlint/intellij/analysis/SonarLintJobManager.java

```diff
@@ -30,6 +30,7 @@
 import com.intellij.util.messages.MessageBus;
 
 import java.util.Collection;
+import java.util.Collections;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.Collections'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)